test/app.test.js

```diff
@@ -0,0 +1,279 @@
+			const errorMessage = await innerDiv.getText();
+
+			const condition = errorMessage.includes("Passwords don't match");
+
+			assert.ok(condition)
+		})
+
+		it('test for signup success message', async function() {
+
+			sendData('username', 'test@email.com', 'test-pass', 'test-pass')
+
+			await driver.sleep(1500)
+
+			const successDiv = driver.findElement(By.className('w-form-done'));
+			const innerDiv = await successDiv.findElement(By.tagName('div'));
+			const successMessage = await innerDiv.getText();
+
+			const condition = successMessage.includes("Thank you! Your submission has been received!");
+
+			assert.ok(condition)
+		});
+	});
+
+	describe('logout test', function() {
+		before(async function(){
+			await driver.get(LOGIN_URL);
+			await driver.sleep(1000)
+			const usernameBox = driver.findElement(By.id('my-muni-Username'));
+			const passwordBox = driver.findElement(By.id('my-muni-Password'));
+
+			usernameBox.sendKeys(VALID_USER);
+			passwordBox.sendKeys(VALID_PASSWORD);
+			const button = driver.findElement(By.className('button form-submit w-button'));
+			button.click();
+
+			await driver.sleep(2500);
+		});
+
+		it('verify logout worked correctly', async function() {
+			const button = driver.findElement(By.className('icon nav-menu__icon'));
+			button.click();
+
+			await driver.sleep(1500);
+
+			const logoutBtn = driver.findElement(By.id('my-muni-logout'));
+			logoutBtn.click();
+
+			await driver.sleep(1500);
+
+			const storage = await driver.executeScript('return window.localStorage');
+			const accessToken = storage.accessToken;
+			const refreshToken = storage.refreshToken;
+		    const condition = accessToken && accessToken.length > 0 && refreshToken && refreshToken.length > 0
+
+		    assert.ok(condition);
+
+
+		});
+	});
+
+	describe('change password while logged in', function() {
+		let password;
+		let oldPassword;
+		let confirmPassword;
+		let button;
+
+		before(async function(){
+			await driver.get(LOGIN_URL);
+			const usernameBox = driver.findElement(By.id('my-muni-Username'));
+			const passwordBox = driver.findElement(By.id('my-muni-Password'));
+
+			usernameBox.sendKeys(VALID_USER);
+			passwordBox.sendKeys(VALID_PASSWORD);
+			const button = driver.findElement(By.className('button form-submit w-button'));
+			button.click();
+
+			await driver.sleep(2000);
+
+			await driver.get(CHANGE_PASSWORD_URL);
+		});
+
+		beforeEach(async function() {
+			oldPassword = driver.findElement(By.id('my-muni-old_password'));
+			password = driver.findElement(By.id('my-muni-password'));
+			confirmPassword = driver.findElement(By.id('my-muni-password_confirm'));
+			button = driver.findElement(By.className('button form-submit w-button'))
+
+			oldPassword.clear();
+			password.clear();
+			confirmPassword.clear();
+		});
+
+		async function sendData(ol, pa, cf){
+			oldPassword.sendKeys(ol);
+			password.sendKeys(pa);
+			confirmPassword.sendKeys(cf);
+
+			button.click();
```

Comment:
depending on this variable being set outside this function leads to very broken code - data should be passed as arguments or fetched from the environment inside functions